WalletWasabi.Fluent/Behaviors/HoldKeyBehavior.cs

```diff
@@ -0,0 +1,56 @@
+using System.Reactive.Disposables;
+using System.Reactive.Linq;
+using Avalonia;
+using Avalonia.Data;
+using Avalonia.Input;
+using Avalonia.VisualTree;
+using ReactiveUI;
+using WalletWasabi.Fluent.Extensions;
+
+namespace WalletWasabi.Fluent.Behaviors;
+
+public class HoldKeyBehavior : AttachedToVisualTreeBehavior<InputElement>
+{
+	public static readonly StyledProperty<Key?> KeyProperty =
+		AvaloniaProperty.Register<HoldKeyBehavior, Key?>(nameof(Key));
+
+	public static readonly StyledProperty<bool> IsKeyPressedProperty = AvaloniaProperty.Register<HoldKeyBehavior, bool>(nameof(IsKeyPressed), defaultBindingMode: BindingMode.TwoWay);
```

Comment:
Can you split property definition into two lines, it really hard to read code beyond the 80 chars columns, especially on lower resolution screens and split window when doing reviews.